src/ingest-pipeline/airflow/dags/scan_and_begin_processing.py

```diff
@@ -233,6 +245,7 @@ def flex_maybe_spawn(**kwargs):
         )
 
     (dag >> t_create_tmpdir >> t_run_validation >>
+     t_maybe_continue >>
```

Comment:
Educational question:

```dag >> t_create_tmpdir >> t_run_validation >> t_maybe_continue >> t_run_md_extract >> ...```

Technically-speaking only, is this specific connection here between `t_maybe_continue` and `t_run_md_extract` required given that `t_maybe_continue` has `'next_op': 'run_md_extract'` in its definition?

Style and clarity are obviously important, but I just wanted to understand the net effect of one versus the other.

Reply:
Yes, it turns out to explicitly needed.  I guess that's a design decision on the part of the Airflow authors.